st2common/bin/st2ctl

```diff
@@ -57,16 +57,27 @@ function validate_in_components() {
 
 function st2start() {
   for COM in ${COMPONENTS}; do
-    service ${COM} start
+    service_manager ${COM} start
   done
 }
 
 function st2stop() {
   for COM in ${COMPONENTS}; do
-    service ${COM} stop
+    service_manager ${COM} stop
   done
 }
 
+function service_manager() {
+  if command -v systemctl 2>/dev/null; then
+    systemctl "${2}" "${1}"
```

Comment:
Wrong assumption. ex. docker container started without systemd, you check binary it exists then you invoke a call and it fails... You *must* check whether systemd is **running**.